test/test_grid.py

```diff
@@ -982,3 +983,12 @@ def test_dual_mesh_mpas(self):
 
         # Assert the faces are the same
         nt.assert_equal(dual.face_node_connectivity.values,  mpas_dual.face_node_connectivity.values)
+
+
+    def test_duplicate(self):
+        uxgrid = ux.open_grid(gridfile_geos)
+        uxgrid.merge_duplicate_node_indices()
```

Comment:
add an assert to check that `compute_dual` fails or produces incorrect stuff without this operation.